iPetCare/Application/Interfaces/IRaceService.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Threading.Tasks;
+using Application.Dtos.Races;
+using Application.Services.Utilities;
+
+namespace Application.Interfaces
+{
+    public interface IRaceService
+    {
+        Task<ServiceResponse<CreateDtoResponse>> CreateAsync(CreateDtoRequest dto);
+        Task<ServiceResponse<GetAllDtoResponse>> GetAllAsync();
+        Task<ServiceResponse<GetDtoResponse>> GetAsync(int raceId);
+        Task<ServiceResponse<PutDtoResponse>> PutAsync(int raceId, PutDtoRequest dto);
+        Task<ServiceResponse<DeleteDtoResponse>> DeleteAsync(int raceId);
```

Comment:
Tak właściwie czy jest potrzeba, żeby zwracać Dtosa w przypadku usuwania? W moim odczuciu to nie, wystarczy tylko status code (czyli zwracalibyśmy Task<ServiceResponse>

Reply:
Nad tym, czy przy usuwaniu wgl jakiś dto jest potrzebny też się zastanawiałam i najpierw myślałam, że nie, a potem stwierdziłam, że może fajnie jakby się wyświetliło użytkownikowi "Rasa xyz została usunięta"

Reply:
Myślę, że to bardziej już kwestia frontendu jak zareaguje na status code 200 :)